### DISCUSSION TITLE
python read_parquet method should accept a path_or_buffer like read_csv.

### DISCUSSION THREAD

**Discussion by publicmatt** [duckdb/duckdb#13915] [Ideas]
I'm using ffspec and I use duckdb like this:

```python
# fs: AbstractFileSystem = ...
with fs.open("bucket-name/file.parquet") as f:
    rel = duckdb.read_parquet(f)
```

This works with csv and json, but not parquet. It would be nice to extend the api to allow paths or file handles/buffers.

**Comment by Tishj** [comment]
You're welcome to make that contribution, <https://github.com/duckdb/duckdb/pull/13040> could serve as a good example to implement this 👍
